client/src/components/BoardGameShowPage.js

```diff
@@ -28,17 +33,51 @@ const BoardGameShowPage = (props) => {
 		}
 	}
 
+	const handleDelete = async () => {
+		try {
+			const response = await fetch(`/api/v1/board-games/${id}`, {
+				method: "DELETE",
+				headers: new Headers({
+					"Content-Type": "application/json"
+				})
+			})
+			if (!response.ok) {
+				if (response.status === 422) {
+					const newErrors = translateServerErrors(body.errors)
+					return setErrors(newErrors)
+				} else {
+					const errorMessage = `${response.status} (${response.statusText})`
+					const error = new Error(errorMessage)
+					throw(error)
+				}
+			} else {
+				props.history.push("/board-games")
```

Comment:
nope, please use the redirect pattern we designated in the week 4 Redirect article....which explicitly states not to do this!